src/app/events/components/evaluation/evaluation-dialog/evaluation-default-grade-dialog.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="modal-header">
+  <h5 class="modal-title">{{ "evaluation.set-default-value" | translate }}</h5>
+</div>
+<div class="modal-body">
+  <div class="dropdown-row">
+    <label for="grade">Note</label>
+    <bkd-select
+      id="grade"
+      [options]="options()"
+      [(value)]="selectedGradeKey"
+      [allowEmpty]="false"
+    ></bkd-select>
+  </div>
+  <p class="info-text">
+    {{ "evaluation.set-default-value-message" | translate }}
+  </p>
+</div>
+<div class="modal-footer">
+  <button type="button" class="btn btn-outline-secondary" (click)="cancel()">
+    {{ "evaluation.cancel" | translate }}
+  </button>
+  <button
+    type="button"
+    class="btn btn-primary"
+    [disabled]="!selectedGrade()"
+    (click)="updateGrades()"
+  >
+    {{ "evaluation.save" | translate }}
+  </button>
```

Comment:
Nice wäre noch, wenn es einen Spinner gäbe während dem Speichern, wie z.B. in der EditAbsencesEditComponent:

```html
        <button
          type="submit"
          class="btn btn-primary ms-2"
          [disabled]="saving$ | async"
        >
          {{ "edit-absences.edit.save" | translate }}
          @if (saving$ | async) {
            <div
              class="spinner-border spinner-border-sm align-middle"
              role="status"
            >
              <span class="visually-hidden">Loading...</span>
            </div>
          }
        </button>
```

Der Saving-State könnte dann von einem Signal vom `EvaluationUpdateService` kommen.

Reply:
Würde ich im Moment so sein lassen.